tools/openmrs_setup

```diff
@@ -71,6 +71,33 @@ mysql -uroot -p <<< "
 "
 echo 'Added MySQL user "openmrs_user" with password "openmrs".'
 
+# Running from Linux or Mac? Set the appropriate path.
+buendia_dir='/usr/share/buendia'
+bin_dir='/usr/bin'
+OS=`uname`
+if [[ "$OS" == 'Darwin' ]]; then
+    buendia_dir='/usr/local/opt/buendia'
+    bin_dir='/usr/local/bin'
+fi
+profiles_dir="$buendia_dir/profiles"
+
+# TODO: remove the need for sudo
+# Create directories and copy scripts
+sudo mkdir -p $profiles_dir
+sudo cp -p tools/profile_validate "$bin_dir/buendia-profile-validate"
+sudo cp -p packages/buendia-utils/data/usr/share/buendia/utils.sh "$buendia_dir/utils.sh"
+
+# In the profile_apply script there is a hardcoded reference to utils.sh. sed will replace this a mac friendly url.
+# TODO: review this after system wide path related changes.
+if [[ "$OS" == 'Darwin' ]]; then
+    sed 's/\/usr\/share/\/usr\/local\/opt/g' tools/profile_apply > "$bin_dir/buendia-profile-apply"
```

Comment:
This execution is throwing `tools/openmrs_setup: line 93: /usr/local/bin/buendia-profile-apply: Permission denied`

Refactoring to `sed 's/\/usr\/share/\/usr\/local\/opt/g' tools/profile_apply > sudo "$bin_dir/buendia-profile-apply"`worked fine.